app.rb

```diff
@@ -0,0 +1,133 @@
+require_relative 'student'
+require_relative 'teacher'
+require_relative 'book'
+require_relative 'rental'
+require_relative 'classroom'
+
+class App
+  def initialize
+    @people = []
+    @books = []
+    @rentals = []
+  end
+
+  def display_all_people
+    @people.each_with_index do |people, index|
+      puts "#{index + 1}) [#{people.title}] Name: #{people.name}, ID: #{people.id}, Age: #{people.age}"
+    end
+  end
+
+  def display_all_books
+    @books.each_with_index do |book, index|
+      puts "#{index + 1}) Title: '#{book.title}', Author: '#{book.author}'"
+    end
+  end
+
+  def create_person
+    personal_data = create_personal_data_person
+    if personal_data[2] == 1
+      print 'Has parent permission [Y/N]: '
+      permission = 'a'
+      while permission != 'Y' && permission != 'N'
+        puts 'Invalid input. Please enter Y or N:'
+        permission = gets.chomp.upcase
+      end
+      parent_permission = permission == 'Y'
+      create_student(persona_data[0], persona_data[1], parent_permission, classroom: 'class')
```

Comment:
- [ ] I got an error when trying to create a student. Instead of `person_data` it should be `personal_data`.
![image](https://github.com/lordksix/library-ruby/assets/45460192/4c35a06c-4885-4dd5-ac6d-e02b4c59b017)

Reply:
Thank you for your observation. The bug has been fix